modules/sildisco/public/metadata.php

```diff
@@ -224,5 +229,5 @@
         exit(0);
     }
 } catch (Exception $exception) {
-    throw new \SimpleSAML\Error\Error('METADATA', $exception);
+    throw new Error\Error('METADATA', $exception);
```

Comment:
I understand the less 'use' statements, but SimpleSAML_Error would be good in my opinion.